<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src/logo.qrc">
    <normaloff>:/logo/calculator-solid.svg</normaloff>:/logo/calculator-solid.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #E6E6E6;
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="main">
   <widget class="QPushButton" name="Input_Equ">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>81</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:rgb(255,255,255);
	border-radius:3px;
	background-color:rgb(0,109,204);
	font-family:Arial;
}
QPushButton:hover
{
	background-color:rgb(0 ,119, 224 );
}
QPushButton:pressed
{
	background-color:rgb(0 ,149 ,255  );
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="BlueButton" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-right: 1px solid #A4A4A4;
	border-bottom:  1px solid #A4A4A4;
	border-bottom-right-radius:3px;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Add">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:3px;
	background-color: #FFFFFF;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Mult">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:3px;
	background-color: #FFFFFF;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Div">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:3px;
	background-color: #FFFFFF;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Sub">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:3px;
	background-color: #FFFFFF;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:1px;
	border-bottom: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-bottom-left-radius:3px;
	border-left: 1px solid #A4A4A4;
	border-bottom: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:1px;
	border-left: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:1px;
	border-right: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:1px;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-top-left-radius:3px;
	border-left: 1px solid #A4A4A4;
	border-top: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-top-right-radius:3px;
	border-top: 1px solid #A4A4A4;
	border-right: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius:1px;
	border-top: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-top-left-radius:3px;
	border-bottom-left-radius:3px;
	border-left: 1px solid #A4A4A4;
	border-top: 1px solid #A4A4A4;
	border-bottom: 1px solid #A4A4A4;
	background-color: #FFFFFF;
	font-family:Arial;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Reset">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFFFFF;
	border-radius:3px;
	background-color: #F78181;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: #F5A9A9;
}
QPushButton:pressed
{
	background-color: #F6CECE;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Neg">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-top-right-radius:3px;
	border-bottom-right-radius:3px;
	background-color: #FFFFFF;
	border-top: 1px solid #A4A4A4;
	border-right: 1px solid #A4A4A4;
	border-bottom: 1px solid #A4A4A4;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: #F2F2F2;
}
QPushButton:pressed
{
	background-color: #BDBDBD;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input_Bksp">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFFFFF;
	border-radius:3px;
	background-color: #FE9A2E;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: #FAAC58;
}
QPushButton:pressed
{
	background-color: #F7BE81;
}</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rt_Last_Res">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #FFFFFF;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    background-color: #6E6E6E;
    font-family: Microsoft YaHei UI;
}</string>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="About">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #79fb79, stop:0.55 #00ffbf, stop:0.98 #aefe58, stop:1 #aefe58);
	color: #FFFFFF;
	border-radius:3px;
	font-family: Microsoft YaHei UI;
}
QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #f8c3c3, stop:0.55 #fef6f6, stop:0.98 #d8f0f7, stop:1 #d8f0f7);
}
QPushButton:pressed
{
	background-color: #39c5bb;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src/logo.qrc">
      <normaloff>:/logo/info-solid.svg</normaloff>:/logo/info-solid.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Textbox_Operand">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black;
	border-radius:3px;
	background-color: #EFFBF2;
	font-family: Microsoft YaHei UI;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Textbox_Input">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: black;
	border-radius:3px;
	background-color: #FFFFFF;
	font-family: Microsoft YaHei UI;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Text_Result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: black;
	border-top-left-radius:3px;
	border-bottom-left-radius:3px;
	background-color: #E0F2F7;
	font-family: Microsoft YaHei UI;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src/logo.qrc"/>
 </resources>
 <connections/>
</ui>
